src/config/framework.config.ts

```diff
@@ -0,0 +1,294 @@
+import * as moment from 'moment'
```

Comment:
I would like to refactor this file, any suggestion will be very much appreciated